st2actions/tests/unit/test_mistral_v2.py

```diff
@@ -264,6 +273,69 @@ def test_launch_workflow_with_auth(self):
             WF1_NAME, workflow_input=workflow_input, env=env)
 
     @mock.patch.object(
+        client.Client, 'authenticate',
+        mock.MagicMock(return_value=(cfg.CONF.mistral.v2_base_url, '123', 'abc', 'xyz')))
+    @mock.patch.object(
+        workflows.WorkflowManager, 'list',
+        mock.MagicMock(return_value=[]))
+    @mock.patch.object(
+        workflows.WorkflowManager, 'get',
+        mock.MagicMock(return_value=WF1))
+    @mock.patch.object(
+        workflows.WorkflowManager, 'create',
+        mock.MagicMock(return_value=[WF1]))
+    @mock.patch.object(
+        executions.ExecutionManager, 'create',
+        mock.MagicMock(return_value=executions.Execution(None, WF1_EXEC)))
+    def test_launch_workflow_with_mistral_auth(self):
```

Comment:
If you could move these tests to test_mistral_v2_auth.py, it would be great. This is > 700 lines file :P.